.gitignore

```diff
@@ -121,9 +121,9 @@ geckodriver.log
 env_file
 
 ### Terraform ###
-.Terraform
+.terraform.lock.hcl
```

Comment:
You should not ignore this file.

Similar to poetry.lock it is used to enforce more specific versions of dependencies.